redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
The time and log sink modules currently have a root 'demo' package - so we should change this across the board.  I suggest we use the same package name as where the code lives and just call it <name of module>Application.  for example  org.springframework.cloud.stream.module.redis.RedisSinkApplication.  Package names in time/log are not good either - 'source' and 'sink'.